pennylane/qnode.py

```diff
@@ -720,6 +720,65 @@ def circuit():
             charset=charset, wire_order=wire_order, show_all_wires=show_all_wires
         )
 
+    @property
+    def specs(self):
+        """Resource information about a quantum circuit.
+
+        Returns:
+            a dictionary listing properties of the qnode

```

Comment:
```suggestion
        dict[str, Union[defaultdict,int]]: dictionaries that contain QNode specifications
```
To make it in line with the tape style?